es/introduction.md

```diff
@@ -94,15 +94,15 @@ Para utilizar el código público existente, hay que especificar en el presupues
 
 ## Cumplimiento de normas o proceso de certificación
 
-La Foundation for Public Code se asegura de que las codebase bajo su tutela o stewardship (y no *in incubation* o *in the attic* ) cumplan con el Estándar para el Código Público. Esto deja claro a los usuarios y colaboradores potenciales que la codebase es de alta calidad, y que las actualizaciones también lo serán.
+La Foundation for Public Code se asegura de que las codebase bajo su tutela o dirección (y no *in incubation* o *in the attic* ) cumplan con el Estándar para el Código Público. Esto deja claro a los usuarios y colaboradores potenciales que la codebase es de alta calidad, y que las actualizaciones también lo serán.
 
-La auditoría realizada por nuestra organización está pensada para complementar los tests automatizados, ya que las máquinas son excelentes para comprobar cosas como la sintaxis y si los resultados se ajustan a las expectativas. Las máquinas no pueden comprobar cosas que están destinadas a los humanos, como comprobar si la documentación es realmente comprensible y accesible en su contexto, si los mensajes de confirmación tienen sentido y si se siguen las directrices de la comunidad.
+La auditoría realizada por nuestra organización está pensada para complementar las pruebas automáticas, ya que las máquinas son excelentes para comprobar cosas como la sintaxis y si los resultados se ajustan a las expectativas. Las máquinas no pueden comprobar cosas que están destinadas a los humanos, como comprobar si la documentación es realmente comprensible y accesible en su contexto, si los mensajes de confirmación tienen sentido y si se siguen las directrices de la comunidad.
 
 La auditoría pone a prueba toda la codebase, incluyendo el código fuente, la política, la documentación y la conversación para comprobar que se cumplen tanto las normas establecidas por nuestra organización como las normas establecidas en la propia codebase.
 
 ### Cómo funciona el proceso
 
-Cada vez que se sugiere una contribución a una codebase -por ejemplo, a través de una solicitud de merge- los [administradores de la codebase](https://about.publiccode.net/roles/) de nuestra organización auditarán la contribución para comprobar que cumple con el Estándar para el Código Público. Las nuevas contribuciones sólo pueden ser adoptadas en la codebase después de haber sido aprobadas conforme con el Estándar para el Código Público, y haber sido revisadas por otro contribuyente.
+Cada vez que se sugiere una contribución a una codebase - por ejemplo, a través de una solicitud de merge - los [administradores de la codebase](https://about.publiccode.net/roles/) de nuestra organización auditarán la contribución para comprobar que cumple con el Estándar para el Código Público. Las nuevas contribuciones sólo pueden ser adoptadas en la codebase después de haber sido aprobadas conforme con el Estándar para el Código Público, y haber sido revisadas por otro contribuyente.
```

Comment:
Spanish doesn't use those spaces in between: https://es.wikipedia.org/wiki/Raya_(puntuaci%C3%B3n)

Can also be substituted by normal parenthesis.